numpy/typing/__init__.py

```diff
@@ -68,13 +68,26 @@
 
 .. code-block:: python
 
-    x = np.array([1, 2])
-    x.dtype = np.bool_
+    >>> x = np.array([1, 2])
+    >>> x.dtype = np.bool_
 
 This sort of mutation is not allowed by the types. Users who want to
 write statically typed code should insted use the `numpy.ndarray.view`
 method to create a view of the array with a different dtype.
 
+dtype
+~~~~~
+
+The ``_DTypeLike`` type tries to avoid creation of dtype objects using
+dictionary of fields like below:
+
+.. code-block:: python
```

Comment:
```suggestion
.. code-block:: python

```

Reply:
Adding an additional empty line should fix the latest CI failure ([ref](https://app.circleci.com/pipelines/github/numpy/numpy/2654/workflows/70c4c38a-0aa0-4894-b8b2-96a76e5c01dc/jobs/14744)).

Reply:
Thanks @BvB93 !